source/features/pr-merge-commit-reviewers.tsx

```diff
@@ -0,0 +1,39 @@
+import select from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import onPrMergePanelOpen from '../github-events/on-pr-merge-panel-open';
+
+function init(): void {
+	const messageField = select('textarea#merge_message_field')!;
+	const reviewers = select.all('span.reviewers-status-icon');
+
+	if (reviewers !== null && !messageField.value.includes('Approved by')) {
```

Comment:
`select.all` always returns an array, no need to check for `null`. Also try to return early whenever possible instead of nesting blocks.